docker/entrypoint.sh

```diff
@@ -85,7 +85,8 @@ case "$1" in
         elif [[ "$TEST_LWD_RPC_CALL" -eq "1" ]]; then
             # Starting at a cached Zebra tip, test a JSON-RPC call to Zebra.
             ls -lh "$ZEBRA_CACHED_STATE_DIR"/*/* || (echo "No $ZEBRA_CACHED_STATE_DIR/*/*"; ls -lhR  "$ZEBRA_CACHED_STATE_DIR" | head -50 || echo "No $ZEBRA_CACHED_STATE_DIR directory")
-            cargo test --locked --release --features "$ENTRYPOINT_FEATURES" --package zebrad --test acceptance -- --nocapture --include-ignored fully_synced_rpc_test
+            # Run both the fully synced RPC test and the subtree snapshot test
+            cargo test --locked --release --features "$ENTRYPOINT_FEATURES" --package zebrad --test acceptance -- --nocapture --include-ignored fully_synced_rpc_
```

Comment:
I'm really sorry about this, but my suggestion was incomplete.

What's happening now is that both tests are running in parallel on the same cached state directory, so one of them panics with a lock file error:
> Message:  Opening database "/var/cache/zebrad-cache/state/v25/mainnet" failed: Error { message: "IO error: While lock file: /var/cache/zebrad-cache/state/v25/mainnet/LOCK: Resource temporarily unavailable" }. Hint: Check if another zebrad process is running. Try changing the state cache_dir in the Zebra config.

https://github.com/ZcashFoundation/zebra/actions/runs/6241663187/job/16945479607?pr=7515#step:9:232

So far I've suggested two different ways to add an extra cached state test, because I thought they would be faster for you. But we've never done it this way before, and I didn't check myself. So that's my mistake.

I will try one last thing in this PR and see if it works. Otherwise I'll open a ticket to document the standard way to add a new job.

Reply:
If CI still isn't passing after my fix, here is the standard test checklist from ticket #7589:

To add a new cached state test job to CI:
- [x] Add a new Rust test and mark it as `#[ignored]`
- [ ] Run that Rust test from a new `elif` block in [entrypoint.sh](https://github.com/ZcashFoundation/zebra/blob/main/docker/entrypoint.sh)
- [ ] Add a new job to `[continous-integration-docker.yml](https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/continous-integration-docker.yml)
- [ ] Add a new patch job to [continous-integration-docker.patch.yml](https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/continous-integration-docker.patch.yml)
- [ ] Check the new test is being run by looking at the CI logs
- [ ] Admin: after the PR merges, add a new branch protection rule for the new job